packages/services/.env.example

```diff
@@ -9,3 +9,6 @@ SEARCH_TYPEAHEAD_HOST=<host for ibm.com search, e.g. https://www-api.ibm.com>
 # Marketing Search
 MARKETING_SEARCH_VERSION=v3
 MARKETING_SEARCH_HOST=<host for ibm.com marketing search, e.g. https://www.ibm.com>
+
+# Translation
+CORS_PROXY=<CORS proxy server for fetching from www.ibm.com, e.g. https://yourproxy.com/>
```

Comment:
Do we need to add this to the `react` `.env.example` file as well? It took me a little while to figure out it was required in the package it's running from to make it work. Part of that was my ignorance too. Maybe that or add note here that `.env` in this file won't be enough to get it running locally.

Reply:
I'll add it to the react .env.example, won't hurt to have it there

Reply:
@photodow added!